test/perf/perf_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package perf
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+type ByteSize int64
+
+const (
+	_           = iota
+	KB ByteSize = 1 << (10 * iota)
+	MB
+	GB
+)
+
+func TestBenchmarkCopyingLargeImageWithinSameRegistryShouldBeFast(t *testing.T) {
```

Comment:
can we add cf-for-k8s test from kbld as a "pressure" test for large number of images? (https://github.com/vmware-tanzu/carvel-kbld/blob/develop/test/e2e/packaging_test.go#L160)

Reply:
I did the manual test for this. Yeah i'm happy to add it to the perf test

Reply:
On a call with @cppforlife and he suggested adding a time assertion. make sure copying a large cf-for-k8s bundle takes less than X time.